src/NRules/NRules.Testing/IRuleVerification.cs

```diff
@@ -1,6 +1,13 @@
 ﻿namespace NRules.Testing;
 
+/// <summary>
+/// Represents specific rule verification
+/// </summary>
 public interface IRuleVerification
 {
+    /// <summary>
+    /// Asserts that rule was fired exact amount of times
```

Comment:
@gritcsenko ".. rule fired exact number of times".
The rule is not fired by someone else, but it itself fires (at least that the terminology I saw in the industry for inference engines).

Reply:
Fixed